src/steamship/agents/tools/video_veneration/did_video_generator_tool.py

```diff
@@ -0,0 +1,57 @@
+"""Tool for generating images."""
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+from steamship.agents.tools import ImageGeneratorTool
+from steamship.utils.repl import ToolREPL
+
+
+class DIDVideoGeneratorOptions(BaseModel):
+    source_url: str
+    """The URL of the source image to be animated."""
+
+    stitch: str
+
+    provider: Optional[str]
+    """The URL of the D-ID driver video. If not provided a driver video will be selected automatically."""
+
+    driver_url: Optional[str]
+    """The URL of the D-ID driver video. If not provided a driver video will be selected automatically."""
+
+    expressions: Optional[List[dict]]
```

Comment:
@EniasCailliau is this the right format for the `expressions` field? 

The `talk` generation API docs made it seem as if this was an array of frame-pegged markers, but that seemed to require knowledge of the frames

Reply:
Yea. It's not ideal. To make it worse they raise exceptions when you referenced non-existing markers.